src/algebra/algebra/basic.lean

```diff
@@ -1224,24 +1225,15 @@ begin
 end
 
 @[simp] lemma pow_lmul_left (a : A) (n : ℕ) :
-  (lmul_left R a)^n = lmul_left R (a ^n) :=
-begin
-  ext b,
-  induction n with n ih,
-  { simp only [linear_map.id_coe, linear_map.one_apply, lmul_left_one, id.def, pow_zero], },
-  { simp only [lmul_left_apply] at ih,
-    simp only [lmul_left_apply, pow_succ, linear_map.mul_apply, ih, mul_assoc], },
-end
+  (lmul_left R a) ^ n = lmul_left R (a ^ n) :=
+((lmul R A).map_pow a n).symm
 
 @[simp] lemma pow_lmul_right (a : A) (n : ℕ) :
-  (lmul_right R a)^n = lmul_right R (a ^n) :=
+  (lmul_right R a) ^ n = lmul_right R (a ^ n) :=
 begin
-  ext b,
-  induction n with n ih,
-  { simp only [linear_map.id_coe, linear_map.one_apply, lmul_right_one, id.def, pow_zero], },
-  { rw lmul_right_apply at ih,
-    rw [pow_succ, pow_succ, linear_map.mul_apply, ih, lmul_right_apply, lmul_right_apply,
-      mul_assoc, ← pow_succ, ← pow_succ'], },
+  apply linear_map.coe_injective,
+  dsimp only, -- TODO[gh-8102]: remove this once `coe_injective` does not introduce junk
```

Comment:
Are you happy to wait for this to be merged first?

Reply:
Absolutely!

How about I cherry pick its commit and make it a dependency and then shorten this proof?

Reply:
I've actually just done this.